conceptnet5/nodes.py

```diff
@@ -38,6 +41,86 @@
 }
 
 
+def standardize_as_list(text, token_filter=None):
```

Comment:
I'll fix the documentation, but I still want to allow for the possibility of a filter besides `english_filter` later.